src/webapi/date.rs

```diff
@@ -17,6 +17,18 @@ impl Date {
             return Date.now();
         ).try_into().unwrap()
     }
+
+	/// The getTimezoneOffset() method returns the time zone difference, in minutes, from UTC to current locale (host system settings).
+	///
+	/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset)
+	pub fn timezone_offset_minutes() -> i32 {
+		let ret = js! {
+			return new Date().getTimezoneOffset();
+		};
+		let ret: i32 = ret.try_into().unwrap_or(0);
+		ret
+	}
+
```

Comment:
1. This needs a link to the IDL in the specs (see other bindings for how it should be done)
2. The name should be `get_timezone_offset` to stay as close as possible to the original JS name.
3. `unwrap_or` is unnecessary; it should simply be `unwrap`
4. This method should take `&self` (you'd have to add `new()` too to be able to construct a new `Date`)

Reply:
I couldn't find an idl for the Date object. The spec linked in the other method was as close as I could find (https://www.ecma-international.org/ecma-262/6.0/#sec-date-constructor).

I didn't realize the intention was to match the API as closely as possible to the JS API for this project. I am in the middle of adding all of the methods on the date object for completeness sake then. I'm assuming that it should also inherit the warts of the JS API. For instance, month being 0-based and the days being 1-based, and calling the get_year method get_full_year because the underlying getYear() isn't Y2K-compliant, and they made a new function called get_full_year.

Also, I'm not a rustacean enough to know what's good names for the various constructors. So far I have:

* new()
* new_with_date(year: i32, month: i32, day: i32)
* new_with_time(year: i32, month: i32, day: i32, hours: i32, minutes: i32, seconds: i32)
* new_with_time_and_millis(year: i32, month: i32, day: i32, hours: i32, minutes: i32, seconds: i32, milliseconds: i32)
* new_with_string(date_string: &str)
* new_with_now(now: f64)

Do these seem good, or should I come up with something else?